eventstream.go

```diff
@@ -221,4 +221,5 @@ func NewClient(prefix, stream string, brokers []string, config ...*BrokerConfig)
 type Client interface {
 	Publish(publishBuilder *PublishBuilder) error
 	Register(subscribeBuilder *SubscribeBuilder) error
+	Unregister(topic, eventName string) error
```

Comment:
not sure that we need a public method to unregister. could be done using context(a common way for golang application)

Reply:
my bad, I actually did it to separate the function to make the code easier to read. I will make this method to be private instead. What do you think?